python/axipcie/_AxiGpuAsyncCore.py

```diff
@@ -57,7 +57,7 @@ def __init__(self,
             bitSize      = 8,
             bitOffset    = 0,
             disp         = '{}',
-            mode         = 'RO',
+            mode         = 'RW',
             pollInterval = 1,
```

Comment:
@mmishra9 Good catch on this being a RW (not RW) variable.  Can you also remove all the `pollInterval = 1,` from these RW variables as well?

Reply:
Done